checks/container.go

```diff
@@ -58,10 +58,10 @@ func (c *ContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Me
 	}
 
 	groupSize := len(ctrList) / cfg.MaxPerMessage
-	if len(ctrList) != cfg.MaxPerMessage {
+	if cfg.MaxPerMessage*groupSize != cfg.MaxPerMessage {
```

Comment:
If `groupSize := len(ctrList) / cfg.MaxPerMessage` and `cfg.MaxPerMessage*groupSize`,
then 
`cfg.MaxPerMessage*groupSize = cfg.MaxPerMessage * len(ctrList) / cfg.MaxPerMessage = len(ctrList)`